projects/train/configs/flow/cbc.yaml

```diff
@@ -20,19 +20,17 @@ model:
   class_path: train.models.flow.FlowModel
   init_args:
     outdir: ${oc.env:AMPLFI_OUTDIR}
-    # add path below when running `trainer.test` to load in a checkpoint
-    checkpoint: null
```

Comment:
Was having `checkpoint = null` breaking things?

Reply:
I think I was using an older config file that didn't have this command. It didn't give an error with this.

Reply:
Got it. If you rebase your branch onto the latest changes from main, that should fix it. 

I'm also noticing you are making this PR from your main branch. In the future you should make a feature branch and submit the PR from there.